docs/tutorial.rst

```diff
@@ -108,139 +112,278 @@ For reproducibility we can also choose set seed for the simulator using the
+.. code-block:: python
 
-.. .. code-block:: python
+    >>> model = stdpopsim.PiecewiseConstantSize(species.population_size)
 
-..     chrom = homo_sapiens.genome.chromosomes["chr22"]
-..     ts = msprime.simulate(
-..         sample_size=10,
-..         mutation_rate=chrom.default_mutation_rate,
-..         recombination_map=chrom.recombination_map("HapmapII_GRCh37"))
+5. Set the number of samples and set the simulation engine.
+In this case we will simulate 10 samples and use the default simulator,
+which is `msprime`. But, you can go crazy with the sample size!
+`msprime` is great at simulating large samples!
 
+.. code-block:: python
 
-.. Demographic models can also be used. For example
+    >>> samples = model.get_samples(10)
+    >>> engine = stdpopsim.get_default_engine()
 
-.. .. code-block:: python
+6. Simulate the model with the contig length and number of samples we defined above.
+We capture the simulation results in a tree sequence object
+(:class:`tskit.TreeSequence`).
 
-..     import stdpopsim
-..     from stdpopsim import homo_sapiens
+.. code-block:: python
 
-..     chrom = homo_sapiens.genome.chromosomes["chr22"]
-..     model = homo_sapiens.GutenkunstThreePopOutOfAfrica()
-..     # One sample each from YRI, CEU and CHB.
-..     samples = [msprime.Sample(population=j, time=0) for j in range(3)]
-..     ts = msprime.simulate(
-..         samples=samples,
-..         recombination_map=chrom.recombination_map(),
-..         mutation_rate=chrom.default_mutation_rate,
-..         **model.asdict())
+    >> ts = engine.simulate(model, contig, samples)
 
-.. (This simulation now has a realistic effective population size,
-.. so will produce thousands of variant sties, but still runs very fast.)
+7. We can now do some simple checks that our simulation worked with
+`tskit
+<https://tskit.readthedocs.io>`__.
 
+.. code-block:: python
 
+    >>> ts.num_samples
+    10
+    >>> ts.num_populations
+    1
+    >>> ts.num_mutations
+    6197
+    >>> ts.num_trees
+    6863
+
+As expected, there are 10 samples in one population. We can also see that 6197 mutations
+and 6863 trees were simulated (since we are not using a seed here, the number of mutations
+and trees will be slightly different for you). Try running the simulation again, and notice
+that the number of samples and populations stays the same, while the number of mutations
+and trees changes.
+
+8. In addition to working directly with the simulated tree squence, we can also output
+other common formats used for population genetics analyses.
+We can use ``tskit`` to convert the tree sequence to a vcf file called "foo.vcf".
+See the tskit documentation (:meth:`tskit.TreeSequence.write_vcf`) for more information.
 
-.. _sec_tutorial_generic_models:
+.. code-block:: python
+
+    >>> with open("foo.vcf", "w") as vcf_file:
+    >>>    ts.write_vcf(vcf_file)
+
+Taking a look at the vcf file, we see something like this:
+
+.. code-block::
```

Comment:
I get an error here, don't you? Changing this to
```
.. code-block:: none
```
fixes the error (which is maybe because of old sphinx version?).

Reply:
it works for me. `none` also works. But, on the RTD looks like it doesn't work. So, @gtsambos can you change that?

Reply:
yup